README.md

```diff
@@ -1,6 +1,8 @@
-# Book Library
+# TODO LIST
 
```

Comment:
## Improve the README 
- [ ] Fix the sscreenshot
- [ ] Fix the live link 
- [ ] Remove commented out information.